Fix cypress config errors in the CI

### Is there an existing issue for this?

- [x] I have searched the existing issues

### Expected Behavior

tests pass after dependency updates

### Current Behavior

they fail

### Error message

see https://github.com/neurobagel/react-bagel/pull/211:


```
Your configFile is invalid: /home/runner/work/react-bagel/react-bagel/cypress.config.ts

It threw an error when required, check the stack trace below:

Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/home/runner/work/react-bagel/react-bagel/node_modules/@cypress/code-coverage/task' imported from /home/runner/work/react-bagel/react-bagel/cypress.config.ts
Did you mean to import "@cypress/code-coverage/task.js"?
    at finalizeResolution (node:internal/modules/esm/resolve:283:11)
    at moduleResolve (node:internal/modules/esm/resolve:952:10)
    at defaultResolve (node:internal/modules/esm/resolve:1188:11)
    at ModuleLoader.defaultResolve (node:internal/modules/esm/loader:642:12)
    at #cachedDefaultResolve (node:internal/modules/esm/loader:591:25)
    at ModuleLoader.getModuleJobForRequire (node:internal/modules/esm/loader:347:53)
    at new ModuleJobSync (node:internal/modules/esm/module_job:333:34)
    at ModuleLoader.importSyncForRequire (node:internal/modules/esm/loader:320:11)
    at loadESMFromCJS (node:internal/modules/cjs/loader:1371:24)
    at Module._compile (node:internal/modules/cjs/loader:1511:5)
    at Module._extensions..js (node:internal/modules/cjs/loader:1[61](https://github.com/neurobagel/react-bagel/actions/runs/14183863366/job/39735471959?pr=212#step:5:62)3:10)
    at Module.load (node:internal/modules/cjs/loader:1275:32)
    at Module._load (node:internal/modules/cjs/loader:1096:12)
    at Module.require (node:internal/modules/cjs/loader:1298:19)
    at require (node:internal/modules/helpers:182:18)
    at loadFile (/home/runner/.cache/Cypress/13.17.0/Cypress/resources/app/node_modules/@packages/server/lib/plugins/child/run_require_async_child.js:89:14)
```

### Environment

- OS:
- Python/Node version:


### How to reproduce

_No response_

### Anything else?

TODO:

- [ ] Understand and fix the CI failure
- [x] Roll out the same fix to all other repos that have used react-bagel as a template!

--- a/cypress.config.ts
+++ b/cypress.config.ts
@@ -1,6 +1,6 @@
 /* eslint-disable import/no-extraneous-dependencies */
 import { defineConfig } from 'cypress';
-import codeCoverageTask from '@cypress/code-coverage/task';
+import codeCoverageTask from '@cypress/code-coverage/task.js';
 
 export default defineConfig({
   e2e: {

